_data/projects/yugabyte.yml

```diff
@@ -0,0 +1,11 @@
+name: Yugabyte DB
+desc: Distributed, high-performance SQL database
+site: https://docs.yugabyte.com/latest/contribute/
+tags:
+  - c++
+  - distributed
+  - database
+  - performance
+upforgrabs:
+  name: good first issue
+  link: https://github.com/YugaByte/yugabyte-db/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22
```

Comment:
@iSignal Please change this to `https://github.com/yugabyte/yugabyte-db/labels/good%20first%20issue`